src/devices/cpu/z80/z80.lst

```diff
@@ -6763,9 +6359,8 @@ edff	# DB   ED
 
 00db	# IN   A,(n)
 	call arg
-	TADR = TDAT8 | (A << 8);
-	call in
-	A = TDAT8; WZ = TADR + 1;
+	call in TDAT8 | (A << 8)
+	A = TDAT8; WZ = (TDAT8 | (A << 8)) + 1; // Check!
```

Comment:
Is this going to work the same way?  The WZ update will use the updated A value now while previously it used the original A value.

Reply:
Good catch. I remember I put this comment somwhere but did't find where. Tests work fine so possibly both are "correct". Let me recheck and confirm

Reply:
Updated. Now it matches original.